rmw_connext_shared_cpp/src/node.cpp

```diff
@@ -53,6 +53,23 @@ create_node(
   // This String_dup is not matched with a String_free because DDS appears to
   // free this automatically.
   participant_qos.participant_name.name = DDS::String_dup(name);
+  // since the participant name is not part of the DDS spec
+  // the node name is also set in the user_data
+  DDS_Long name_length = strlen(name);
```

Comment:
This generates a [warning on Windows](http://ci.ros2.org/job/ci_windows/4009/warnings41Result/new/)

Reply:
Fixed.